### ISSUE TITLE
[Question] Why do we need two ways of mocking items?

### ISSUE THREAD

**Issue by vadimzozulya** [smarchenko/SitecoreDI.NSubstitute.Helper#2]
Why do we `FakeUtil.FakeItem();` and `new FakeItem();`?

**Comment by smarchenko** [comment]
FakeItem class is much more powerful one. The plan is complete with unit testing of this class and add some more methods.
Once it is ready, FakeUtil will not be necessary. 
At the same time, I am still not sure whether I should remove this class, It still can be useful for cases when people do not want, for some reason, to use FakeItem. Also, the static helper with a number of tiny methods helps to learn how to do simple things. Probably these methods should be moved to FakeItem class and marked as protected virtual. What do you think?

**Comment by vadimzozulya** [comment]
Agree that number of tiny methods much more understandable so I'd convert FakeUtil to extension methods for FakeItem.

Another questions : 
What is a goal of this repo?
Is there any plans to publish nuget package so anybody can install it and use mock items?
If you don't have such plans, I guess tests for each case are much more clear than FakeUtil or/and FakeItem.

**Comment by smarchenko** [comment]
Yes, the plan is to build a nuget package and open it for downloading.

**Comment by vadimzozulya** [comment]
So in this case I believe it will be much more convenient to have single FakeItem class with all these methods, and probably more complex extension methods (for instance  `AsMediaItem`, `AsDeviceItem`, etc. which will set templates and all necessary fields).
